integration/tests/basic/rpc/test_rpc_neon_estimate_scheduled_gas.py

```diff
@@ -343,30 +473,100 @@ def test_estimate_with_preparatory_solana_transactions(
         assert_fields_are_hex(resp, ["chainId", "maxFeePerGas", "maxPriorityFeePerGas", "nonce", "treasuryIndex"])
 
     def test_estimate_transfer_trx_without_approval_in_preparatory_sol_trx_list(
```

Comment:
please leave this test as it was 
I added it to check dependences between sol transactions and neon transaction
it is looks like previous one but without prepatory sol trx part and it shows that for that transaction we need some solana part for estimation 
Now you got error because you use neon_user_no_sols or something else, the reason has nothing common with the test name